java/ql/test/library-tests/frameworks/apache-commons-lang3/PairTest.java

```diff
@@ -0,0 +1,112 @@
+import org.apache.commons.lang3.tuple.Pair;
+import org.apache.commons.lang3.tuple.ImmutablePair;
+import org.apache.commons.lang3.tuple.MutablePair;
+
+class PairTest {
+    String taint() { return "tainted"; }
+
+    private static class IntSource {
+      static int taint() { return 0; }
+    }
+
+    void sink(Object o) {}
+
+    void test() throws Exception {
```

Comment:
This seems to be missing a test for `Pair.of(left, right)`
Might be good to add it for completeness?